win32/build/mkico.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Not sure if we need a shell script. Developers who want to (re-)create the icon, probably also want to test it, so I'd assume they have a cmd.exe available anyway. On the other hand, it shouldn't hurt to have a shell script.